view/frontend/templates/reviews/title.phtml

```diff
@@ -1,3 +1,43 @@
 <span id='js-lipscore-reviews-tab'>
     <?php echo $block->escapeHtml(__('Reviews')) ?><span id='js-lipscore-reviews-tab-count' style='display: none;'>&nbsp;(<span class='lipscore-review-count' <?php echo $block->getLsProductAttrs() ?>></span>)</span>
 </span>
+<script>
+
+function wcLipscoreInit () {
+    var tabSelector = document.getElementById('js-lipscore-reviews-tab');
+    if (tabSelector && tabSelector.innerHTML != '') {
+        lipscore.on('review-count-set', function (data){
+            if (data.value > 0) {
+                document.getElementById('js-lipscore-reviews-tab-count').style.display = 'block';
+            }
+        });
+
+        lipscore.on('review-count-link-clicked', function (data){
+            var el = document.getElementById('js-lipscore-reviews-tab').parentNode;
```

Comment:
I think we can reuse `tabSelector` here. If it's possible  we will avoid code duplication.

I'm not sure because we tabSelector below with different selector and how JS will handle vars and scopes in this case.
So If you decide to reuse a variable I would suggest to rename vars to something like `reviewsTabSelector` and `qaTabSelector` 😃